### PR TITLE
Map 1795

### PR DESCRIPTION
**Pull request by darrenwiens**
npm install
npm install –g grunt-cli
grunt

Can test either src or deploy, for example:
http://localhost:8000/src/demo.html?appid=10d52161999d4b65bb464549d8978408
http://localhost:8000/deploy/demo.html?appid=feae8ba75ec14bd5a3ba33586d62b2e1

More appids available at: https://storymaps.arcgis.com/en/app-list/map-series/gallery-series/#s=0&md=storymaps-apps:map-series

### CODE REVIEW

File: src/app/main-config.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ function defineDojoConfig()
 		aliases: [
 			['text', 'lib-build/text'],
 			['underscore', 'lib-build/lodash'],
-            ['maptiks', '//cdn.maptiks.com/esri3/mapWrapper.js']
+            ['maptiks', '//cdn-dev.maptiks.com/esri3/mapWrapper.js']
```
Comment (aspcanada) [inline src/app/main-config.js]: keep this using prod in the repo, and use circle to deploy the correct version.

File: src/app/storymaps/tpl/core/Maptiks.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+define(["dojo/topic"],
+  function(topic){
+    topic.subscribe("story-loaded-map", function(){
+      require(['maptiks'], function (mapWrapper) {
+        if (!app.map.maptiks) {
+          var container = app.map.container; // the current map div
+          var maptiksSet = app.data.getWebAppData().getMaptiks().hasOwnProperty('maptiksTrackcode');
+          var maptiksTrackcode = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksTrackcode : 'c311cf16-ad79-42b1-97f9-f433be6c8b00'; // replace with dev trackcode
+          var maptiksId = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksId : "demo";
```
Comment (aspcanada) [inline src/app/storymaps/tpl/core/Maptiks.js]: I don't want to see these in the code that all our templates will use. Can this belong on the demo.html page?

**Review by aspcanada — CHANGES_REQUESTED** [review]

File: src/app/storymaps/tpl/core/Maptiks.js (hunk 2 above)
Comment (darrenwiens) [reply src/app/storymaps/tpl/core/Maptiks.js]: I don't think I can send that out from demo.html. I could work that into the "story-loaded-map" topic (or a custom topic), if that's preferred. Is the idea to have a single generic Maptiks.js that will be accessed by all templates?

File: src/app/main-config.js (hunk 1 above)
Comment (darrenwiens) [reply src/app/main-config.js]: done

**Review by aspcanada — APPROVED** [review]

**Pull request merged**